tests/phpunit/payment-methods/test-class-wc-stripe-upe-payment-gateway.php

```diff
@@ -191,6 +194,8 @@ public function tear_down() {
 		delete_option( WC_Stripe_Feature_Flags::LPM_ACSS_FEATURE_FLAG_NAME );
 		delete_option( WC_Stripe_Feature_Flags::LPM_BACS_FEATURE_FLAG_NAME );
 		delete_option( WC_Stripe_Feature_Flags::AMAZON_PAY_FEATURE_FLAG_NAME );
+
```

Comment:
Nit: Should we remove this line break to keep this `delete_option()` next to the others?